anyway/infographics_utils.py

```diff
@@ -720,7 +719,7 @@ def create_infographics_data(news_flash_id, number_of_years_ago):
             start_time=start_time,
             end_time=end_time,
         ),
-        text={"title": "נפגעים בתאונות במקטע " + location_info["road_segment_name"]},
+        text={"title": _("Accidents\' injuries in section {}").format(location_info['road_segment_name'])},
```

Comment:
```
>>> location_info = {"road_segment_name": "aaa"}
>>> text={"title": _("Accidents\' injuries in section {}").format(location_info['road_segment_name'])},
>>> text[0]["title"]
"Accidents' injuries in section aaa"
```
Don't think that's what you wanted

Reply:
What do you mean?

Reply:
the \` is mis-placed, but what do you think about changing it to :
`f"Injured in accidents in section { location_info['road_segment_name'] }"` ?

Reply:
Will do.